cpp/src/arrow/io/buffered_test.cc

```diff
@@ -293,6 +293,15 @@ TEST_F(TestBufferedOutputStream, Tell) {
 
   ASSERT_OK(buffered_->Close());
 
+  // write long bytes after raw_pos is cached
+  const std::string basic_string = "1234568790";
+  OpenBuffered(/*buffer_size=*/3, false);
+  AssertTell(0);
+  ASSERT_OK(buffered_->Write(basic_string.data(), 5));
```

Comment:
Is `buffered_->Write("1234567890", 5)` enough for the test?

Reply:
done